__tests__/components/App.test.jsx

```diff
@@ -35,4 +35,11 @@ describe('First React component test with Enzyme', () => {
   it('should render photo gallery', () => {
     expect(wrapper.find('PhotoGallery')).toHaveLength(1);
   });
+
```

Comment:
I need to take a more in depth look at your testing suite for this component.  I haven't yet been able to figure how to test my modals and it seems like you figured it out.